pom.xml

```diff
@@ -231,7 +231,7 @@
         <xmlSecurityVersion>1.5.8</xmlSecurityVersion>
         <xstreamVersion>1.4.7</xstreamVersion>
         <xaPoolVersion>1.5.0</xaPoolVersion>
-        <muleCommonVersion>3.8.0-SNAPSHOT</muleCommonVersion>
```

Comment:
No I don't if so can you make the change. 3.8.0-SNAPSHOT was removed by the SNAPSHOT cleaner